<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditStructureForm</class>
 <widget class="QDialog" name="EditStructureForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>211</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Change the Assembly Structure</string>
  </property>
  <widget class="QLabel" name="new_assy_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>150</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>To New Assembly</string>
   </property>
  </widget>
  <widget class="QLabel" name="old_assy_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>150</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>From Old Assembly</string>
   </property>
  </widget>
  <widget class="ErrorFrame" name="old_assy_frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>35</y>
     <width>340</width>
     <height>40</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <widget class="ComboBox" name="old_assy_combo">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>2</y>
      <width>336</width>
      <height>36</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Required: Enter the existing Assembly to change, 1 to 15 Characters</string>
    </property>
   </widget>
  </widget>
  <widget class="ErrorFrame" name="new_assy_frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>340</width>
     <height>40</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <widget class="LineEdit" name="new_assy_edit">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>2</y>
      <width>336</width>
      <height>36</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Required: Enter the new Assembly, 1 to 15 Characters</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="save_done_button">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>160</y>
     <width>90</width>
     <height>36</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Save the current item, then close the form</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save / Done</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="close_button">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>90</width>
     <height>36</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Close the form, optionally saving any changed information</string>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save_new_button">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>160</y>
     <width>90</width>
     <height>36</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Save the current item, then clear the form</string>
   </property>
   <property name="text">
    <string>Save / New</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ErrorFrame</class>
   <extends>QFrame</extends>
   <header>lbk_library/gui/error_frame</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ComboBox</class>
   <extends>QComboBox</extends>
   <header>lbk_library/gui/combo_box</header>
  </customwidget>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>lbk_library/gui/line_edit</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>old_assy_combo</tabstop>
  <tabstop>new_assy_edit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
